### PR TITLE
About page amends

### PR DESCRIPTION
**Pull request by davidrapson**
Updates copy with latest and amends and adds links in.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -191,26 +191,29 @@ toplevel:
...
         linkText: "Discover our strategic framework: People in the Lead"
-        linkHref: #
+        linkHref: "/about-big/strategic-framework"
         imageCaption: Dawn Austwick, CEO
-    outro: We are the largest distributor of National Lottery funding,
-      established as a non-departmental public body by an Act of Parliament.
-      Our work is divided into five portfolios, which cover funding across England,
-      Northern Ireland, Wales and Scotland, and the UK as a whole.
+    outro: We’re the largest distributor of National Lottery funding,
+      established as a non- departmental public body by an Act of Parliament.
```
Comment (mattandrews) [inline config/locales/en.yml]: Do we need a space here after `non-`?

File: config/locales/en.yml [hunk 2]
```diff
@@ -191,26 +191,29 @@ toplevel:
...
+      Our work is divided into five major portfolios, which covering funding across
```
Comment (mattandrews) [inline config/locales/en.yml]: Should this be "cover"?

File: config/locales/en.yml [hunk 3]
```diff
@@ -191,26 +191,29 @@ toplevel:
...
+      England, Northern Ireland, Wales and, Scotland, and the UK as a whole.
```
Comment (mattandrews) [inline config/locales/en.yml]: No need for comma after "and"

File: config/locales/en.yml (hunk 2 above)
Comment (davidrapson) [reply config/locales/en.yml]: Just checked the doc, and I think so. I must've picked up some track-changes leftovers.

File: config/locales/en.yml (hunk 3 above)
Comment (davidrapson) [reply config/locales/en.yml]: Indeed

**Comment by davidrapson** [comment]
@mattandrews Mind giving this another look, I've added a few more bits.

**Pull request merged**